### PR TITLE
(closed without merging)

### PR DESCRIPTION
**Pull request by AjimenezDCL**
Was:

### CODE REVIEW

**Review by eordano — APPROVED** [review]

**Review by eordano — APPROVED** [review]

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs [hunk 1]
```diff
@@ -24,11 +24,17 @@ public void Awake()
 
     public void UpdateMinimapSceneInformation(string scenesInfoJson)
     {
+        Debug.Log("received information");
+        Debug.Log(scenesInfoJson);
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs]: These should be removed or encapsulated in `#if VERBOSE`

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs [hunk 2]
```diff
@@ -24,11 +24,17 @@ public void Awake()
...
+
+        //var scenesInfo = Newtonsoft.Json.JsonConvert.DeserializeObject<MinimapSceneInfo[]>(scenesInfoJson);
         var scenesInfo = Utils.ParseJsonArray<MinimapSceneInfo[]>(scenesInfoJson);
+        Debug.Log("parsed information");
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs]: See above

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs [hunk 3]
```diff
@@ -24,11 +24,17 @@ public void Awake()
...
         foreach (var scene in scenesInfo)
         {
             foreach (var parcel in scene.parcels)
             {
+                Debug.Log($"Set Tile: \n {parcel.x},{parcel.y}  {scene.type} {scene.name}");
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs:41]: See above

File: unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs [hunk 4]
```diff
@@ -24,11 +24,17 @@ public void Awake()
 
     public void UpdateMinimapSceneInformation(string scenesInfoJson)
     {
+        Debug.Log("received information");
+        Debug.Log(scenesInfoJson);
+
+        //var scenesInfo = Newtonsoft.Json.JsonConvert.DeserializeObject<MinimapSceneInfo[]>(scenesInfoJson);
```
Comment (javiermaria) [inline unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Minimap/MinimapMetadataController.cs]: Commented code yada yada

**Comment by AjimenezDCL** [comment]
Just forget about this PR, i'm doing black magic trying to understand the underlying problem. Sorry for the misunderstanding!

**Comment by AjimenezDCL** [comment]
After researching we cannot implement a quickly hotfix. I'm going to close this PR.

**Pull request closed without merging**